daemon/service.go

```diff
@@ -41,6 +42,12 @@ func (s *ServiceV2) Parse(rctx xcontext.Context, req *protocol2.ParseRequest) (r
 		return resp, nil
 	}
 
+	if !utf8.ValidString(req.Content) {
+		err := ErrInvalidEncoding.New()
+		logrus.Debugf("parse v2: %s", err)
```

Comment:
To assist the debugging here and below, do you think it could be beneficial to include a bit of the "context" of this request inside the error? E.g the file name or something similar that would later help to identify the failure source